packages/opentelemetry-instrumentation-togetherai/opentelemetry/instrumentation/togetherai/utils.py

```diff
@@ -0,0 +1,25 @@
+import logging
+from opentelemetry.instrumentation.togetherai.config import Config
+
+
+def dont_throw(func):
+    """
+    A decorator that wraps the passed in function and logs exceptions instead of throwing them.
+
+    @param func: The function to wrap
+    @return: The wrapper function
+    """
+    # Obtain a logger specific to the function's module
+    logger = logging.getLogger(func.__module__)
+
+    def wrapper(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except Exception as e:
+            logger.debug(
+                "OpenLLMetry failed to trace in %s, error: %s", func.__name__, str(e)
```

Comment:
```suggestion
                "OpenLLMetry failed to trace in %s, error: %s", func.__name__, traceback.format_exc()
```
With `import traceback` at the top